pkg/plugin/processor/builtin/impl/clone.go

```diff
@@ -0,0 +1,89 @@
+// Copyright © 2025 Meroxa, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:generate paramgen -output=clone_paramgen.go cloneConfig
+
+package impl
+
+import (
+	"context"
+	"strconv"
+
+	"github.com/conduitio/conduit-commons/config"
+	"github.com/conduitio/conduit-commons/opencdc"
+	sdk "github.com/conduitio/conduit-processor-sdk"
+	"github.com/conduitio/conduit/pkg/foundation/cerrors"
+	"github.com/conduitio/conduit/pkg/foundation/log"
+)
+
+type CloneProcessor struct {
+	sdk.UnimplementedProcessor
+
+	config cloneConfig
+}
+
+func NewCloneProcessor(log.CtxLogger) *CloneProcessor { return &CloneProcessor{} }
+
+type cloneConfig struct {
+	// The number of records after cloning (e.g. if count is 3, the processor
+	// will output 3 records for every input record).
+	Count int `json:"count" validate:"required,gt=1"`
```

Comment:
(If `Count` changes)

```suggestion
	Count int `json:"count" validate:"required,gt=0"`
```